### PR TITLE
LPS-198524 Fix Ai creator module keys

### PR DESCRIPTION
**Pull request by Stephy6**
https://liferay.atlassian.net/browse/LPS-198524

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/aicreator/plugin.js [hunk 1]
```diff
@@ -136,7 +136,7 @@
 				button.classList.add('lfr-portal-tooltip');
 
 				button.setAttribute(
-					'data-title',
```
Comment (victorg1991) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/aicreator/plugin.js:139]: Can we also set the aria-label to this value? For this we probably need to remove the already present aria-labeledby :)

**Comment by victorg1991** [comment]
Niice 👏

**Comment by Stephy6** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**